dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -131,6 +129,13 @@ RUN pip install ansible==2.8.7
 
 RUN pip install pysubnettree
 
+# Install pytest-ansible module
+RUN git clone https://github.com/ansible/pytest-ansible.git \
+    && cd pytest-ansible \
```

Comment:
can you checkout a specific commit it? otherwise, this will consistently pulling the latest master from pytest-ansible which might break the test in some cases in the future.

Reply:
Made changes to checkout the specific commit in pytest-ansible to support 'become' and 'become_user' parameters